doc/builtin_entities.txt

```diff
@@ -0,0 +1,96 @@
+# Builtin Entities
+Minetest registers two entities by default: Falling nodes and dropped items.
+This document describes how they behave and what you can do with them.
+
+## Falling node (`__builtin:falling_node`)
+
+Is created when a node with the special group `falling_node=1` starts to fall.
+Falling nodes can also be created artificially with
+`minetest.spawn_falling_node`.
+
+Spawning a falling node with `/spawnentity` won't work. 
+
+Behaviour:
+
+* Falls down in a straight line (gravity = `movement_gravity` setting)
+* It collides with physical objects and `walkable` nodes
+* If the node group `float=1` is set, it also collides with liquid nodes
+* When it hits a solid (=`walkable`) node, it will try to place itself as a
+  node, replacing the node above
+    * If the destination node is not `buildable_to`, the *destination node*
+      drops as an item
+    * If the falling node fails to place itself, it drops as an item
+
+### Entity fields
+
+* `set_node(self, node, meta)`: Call this function to set the node type that
+  is falling. `node` and `meta` are explained below. The `meta` argument
+  is optional.
+* `node`: Node table of the node (`name`, `param1`, `param2`) that this
+  entity represents. Read-only.
+* `meta`: Node metadata of the falling node. Will be used when the falling
+  nodes tries to place itself as a node. Read-only.
+
+### Rendering / supported nodes
+
+Falling nodes try their best to look as close as possible to the original node.
+This works for most drawtypes, but there are limitations.
+Falling nodes use the `item` entity visual for most nodes.
+
+Supported drawtypes:
+
+* `normal`
+* `signlike`
+* `torchlike`
+* `nodebox`
+* `raillike`
+* `glasslike`
+* `glasslike_framed`
+* `glasslike_framed_optional`
+* `allfaces`
+* `allfaces_optional`
+* `firelike`
+* `mesh`
+* `fencelike`
+* `liquid`
+* `airlike` (not pointable)
+
+Other drawtypes still kinda work, but they might look weird.
+
+Supported `paramtype2`s:
+
+* `wallmounted`
+* `facedir`
+* `colorwallmounted`
+* `colorfacedir`
+* `color`
+
+## Dropped item stack (`__builtin:item`)
+
+This is an item stack in a collectable form. It appears when a player drops an
+item, when a node drops as an item and on various other events.
```

Comment:
Why "various other events"?
The engine only has player drops item, attached nodes and `add_item`.